apps/ledger-live-mobile/e2e/specs/receive/bitcoin.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { device } from "detox";
+import { loadBleState, loadConfig } from "../../bridge/server";
+import PortfolioPage from "../../models/wallet/portfolioPage";
+import ReceivePage from "../../models/receive";
+import DeviceAction from "../../models/DeviceAction";
+import { DeviceModelId } from "@ledgerhq/devices";
+import { tapByText, waitForElementById } from "../../helpers";
+
+let portfolioPage: PortfolioPage;
+let receivePage: ReceivePage;
+let deviceAction: DeviceAction;
+
+const knownDevice = {
+  name: "Nano X de test",
+  id: "mock_1",
+  modelId: DeviceModelId.nanoX,
+};
+
+describe("Bitcoin 2 account", () => {
+  beforeAll(async () => {
+    loadConfig("onboardingcompleted", true);
+    loadBleState({ knownDevices: [knownDevice] });
+
+    portfolioPage = new PortfolioPage();
+    deviceAction = new DeviceAction(knownDevice);
+    receivePage = new ReceivePage();
+
+    await portfolioPage.waitForPortfolioPageToLoad();
+  });
+
```

Comment:
I would add an afterAll reenabling the detox synchronisation just to make sure it doesn't affect the other tests